packages/react-property/scripts/build-svg.js

```diff
@@ -0,0 +1,60 @@
+const fs = require('fs');
+const path = require('path');
+const DOMProperty = require('react-dom/lib/DOMProperty');
+const SVGDOMPropertyConfig = require('react-dom/lib/SVGDOMPropertyConfig');
+const { LIB_DIR, SVG_DIR } = require('./constants');
+
+/**
+ * Creates the DOM property map via injection.
+ *
+ * @see https://github.com/facebook/react/blob/15-stable/src/renderers/dom/shared/DOMProperty.js#L57
+ */
+DOMProperty.injection.injectDOMPropertyConfig(SVGDOMPropertyConfig);
+
+// `autofocus` is removed since it's not an applicable attribute for SVG elements
+delete DOMProperty.getPossibleStandardName.autofocus;
+
+/**
+ * Create output directories (if it does not exist).
+ */
+try {
+  fs.mkdirSync(LIB_DIR);
+} catch (error) {
+  // throw error;
```

Comment:
as above, we could handle this error if it is not expected
```
if (error.code !== 'EEXIST') throw error;
```

Reply:
1566b2f